src/script/components/ZoomableImage/ZoomableImage.tsx

```diff
@@ -0,0 +1,128 @@
+/*
+ * Wire
+ * Copyright (C) 2024 Wire Swiss GmbH
+ *
+ * This program is free software: you can redistribute it and/or modify
+ * it under the terms of the GNU General Public License as published by
+ * the Free Software Foundation, either version 3 of the License, or
+ * (at your option) any later version.
+ *
+ * This program is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE. See the
+ * GNU General Public License for more details.
+ *
+ * You should have received a copy of the GNU General Public License
+ * along with this program. If not, see http://www.gnu.org/licenses/.
+ *
+ */
+
+import React, {useEffect, useRef, useState} from 'react';
+
+import cx from 'classnames';
+
+const ZOOM_IMAGE_SCALE = 3;
+
+interface ZoomableImageProps {
+  src: string;
+  id: string;
+  alt?: string;
+  className?: string;
+}
+
+export const ZoomableImage = ({src, id, alt = '', className = '', ...props}: ZoomableImageProps) => {
```

Comment:
```suggestion
export const ZoomableImage = (props: ZoomableImageProps) => {
```

you don't need to destructure here, right?

Reply:
<img width="857" alt="image" src="https://github.com/wireapp/wire-webapp/assets/13432884/34dc2213-1a5f-4a72-946f-6cc951233aec">

I'm getting this while not pass alt to the image as prop

Reply:
Also I need id for a key, to rerender image to avoid zoom in next/prev image

Reply:
you can do `useId` from react, right? I don't think it's a good idea to use an html `id` as a key, they have 2 distinct roles they should not be confused 

Try with the suggestion here, it should fix the warning you see  https://github.com/wireapp/wire-webapp/pull/17208/files/757c10cbb24912667584a7ddcbca5b4ec5a2a138#r1549687274

Reply:
Ahh.. You're right with useId.

This extends doesn't help